tests/test_models/test_task_modules/test_assigners/test_batch_atss_assigner.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from unittest import TestCase
+
+import torch
+
+from mmyolo.models.task_modules.assigners import BatchATSSAssigner
+
+
+class TestBatchATSSAssigner(TestCase):
```

Comment:
Unit tests should use fewer resources. so many parameters can be changed to smaller